tests/tools/test_dist_part.py

```diff
@@ -152,11 +152,12 @@ def rand_edges(num_src, num_dst, num_edges):
                 assert feat_array.shape[0] == num_edges[etype] // num_chunks
 
         # Step1: graph partition
+        num_parts = 2
```

Comment:
Let's add more tests instead of having one test case. Can you add some corner cases too? e.g., num_parts=1, or num_chunks=1, or large num_chunks but small num_parts, or num_chunks is not divisible by num_parts, etc.